Sources/iosDesignTokens/todaytix/Tokens.swift

```diff
@@ -0,0 +1,282 @@
+
+//
+// Tokens.swift
+//
+
+// Do not edit directly
+// Generated on Thu, 10 Nov 2022 19:19:40 GMT
+
+
+import UIKit
+
+public enum Tokens {
+    public static let borderRadiusLg = CGFloat(8)
+    public static let borderRadiusSm = CGFloat(4)
+    public static let borderRadiusXl = CGFloat(16)
+    public static let borderWidthLg = CGFloat(8)
+    public static let borderWidthMd = CGFloat(4)
+    public static let borderWidthNone = CGFloat(0)
+    public static let borderWidthSm = CGFloat(2)
+    public static let borderWidthXs = CGFloat(1)
+    public static let coreBasic0 = UIColor(red: 1.000, green: 1.000, blue: 1.000, alpha: 1)
```

Comment:
3 decimals of precision? Whoa D:

Reply:
that's what was generated. Is that not ok? I think I can change that in the script but I'm not 100% sure